airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/config.py

```diff
@@ -91,11 +95,13 @@ class BasicReadTestConfig(BaseConfig):
     configured_catalog_path: Optional[str] = configured_catalog_path
     validate_output_from_all_streams: bool = Field(False, description="Verify that all streams have records")
     expect_records: Optional[ExpectedRecordsConfig] = Field(description="Expected records from the read")
+    timeout: int = timeout
```

Comment:
what is the unit of time? :) can it be reflected in the name e.g: `timeout_seconds`

Reply:
it is possible to have duration values in one of the two formats:
```
timedelta fields can be:

timedelta, existing timedelta object
int or float, assumed as seconds
str, following formats work:

[-][DD ][HH:MM]SS[.ffffff]
[±]P[DD]DT[HH]H[MM]M[SS]S (ISO 8601 format for timedelta)
```

https://pydantic-docs.helpmanual.io/usage/types/#datetime-types